client/src/components/modal-forms/PhaseDialog.js

```diff
@@ -15,18 +18,44 @@ const useStyles = makeStyles(() => ({
   formRow: {
     gap: '25px',
   },
+  list: {
+    overflow: 'scroll',
+    maxHeight: '250px',
+  },
+  listItem: {
+    fontSize: '14px',
+    paddingTop: '0',
+    paddingBottom: '0',
+  },
 }));
 
-export const PhaseDialog = ({ onSubmit, onClose, open, content, isNew }) => {
+export const PhaseDialog = ({ onSubmit, onClose, open, content, isNew = false }) => {
   const { openToast } = useToast();
   const classes = useStyles();
+  const [phases, setPhases] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    if (loading) {
```

Comment:
I don't think the name `loading` is very clear here. Just sounding it out, it sounds like "if we're currently loading, then fetch data".

In DataTables, `isPendingRequests` is used I think. May want to follow

Reply:
updated!